### ISSUE TITLE
New Slack Update 3.0.5 breaks this hack

### ISSUE THREAD

**Issue by jimkiely-brainiacdevs** [widget-/slack-black-theme#43]
The new slack 3.0.5 update breaks this hack in Window 10.

I have the readme code in the ssb-interop.js file.  This worked for the past update, but not in 3.0.5.  Anyone else having the same issue?

**Comment by Kredns** [comment]
Yep, I can confirm this as well. @widget- have you been able to reproduce this also?

**Comment by spinegar** [comment]
On mac it seemed to revert my text color. Using a low contract theme made it very difficult to read. I ended up modifying the background css to be a bit lighter for now.

**Comment by yshinkarev** [comment]
Confirm. Xubuntu 16.04 amd64. Ignore theme, like nothing add to index.js

**Comment by nektobit** [comment]
Confirm, win 10

**Comment by david1602** [comment]
Have you tried using [this](https://github.com/widget-/slack-black-theme/issues/39#issuecomment-356698404)?
I am definitely using 3.0.5 and it's working on Windows 10.

**Comment by yshinkarev** [comment]
@david1602 thanks, it's solve issue on my system.

**Comment by Jeremydavisvt** [comment]
@david1602 that isn't working on 3.0.5 on OSX

**Comment by vladiibine** [comment]
For me, on ubuntu 16.04, with 3.0.5, putting the code in ssb-interop.js works.

It's still not nice if I am signed into 2 accounts, but that's another issue

**Comment by Azzuen1** [comment]
I can confirm this as well

**Comment by antok87** [comment]
Dark theme is working ok but text on attachments is black and not readible, any idea how to fix that?

**Comment by Bterrell721** [comment]
Has anyone managed to create a fix for this theme that brings back full css control (text colors) and corrects sidebar overlap issues ("jump to" search overlaying "All Unreads" link)?  

Currently using slack 3.1.0 beta desktop app on Windows 10.  The update from @david1602 referenced above brings back functionality, but the text is stuck on grey and doesn't respect any font color choices.

**Comment by brantleyr** [comment]
I guess the CSS attributes and classes have changed a bit on the actual web app. You will need to apply new class overrides for things that you would like. For example to apply changes to the body text you will need new custom CSS in index.js.

`div.c-message__content { color:#fff !important; }
div.c-message__content span.c-message__body { color:#fff !important; }
div.c-message__content div.c-message__content_header { color:#fff !important; }
div.c-message__content div.c-message__content_header span.c-message__sender a.c-message__sender_link { color:#fff !important; }
div.c-message__content div.c-message__content_header a.c-timestamp { color:#fff !important; }
div.c-message__content div.c-message__content_header a.c-timestamp span.c-timestamp__label { color:#fff !important; }`

Adding the above will make all the body text white. You can find out new classes or other classes you might need by going to the web app in your browser, right clicking what you want, inspecting source, then adding a CSS override as above.

**Comment by brantleyr** [comment]
Seems to be a duplicate of: https://github.com/widget-/slack-black-theme/issues/39

**Comment by bnayagrawal** [comment]
I am using slack on KDE. I applied this hack, It ain't working.

**Comment by seanpianka** [comment]
@bnayagrawal Confirmed that the updated changes to slack have broken this modification (applying changes to either index.js or ssb-interop.js still ends up w/ the chat background being white on white text).

**Comment by jrausch12** [comment]
@seanpianka Also confirming that as well on my end with 3.1.1

**Comment by d-fay** [comment]
@seanpianka and @jrausch12 this is because new CSS elements have been introduced that also need to be colored. The code you are adding to these files is not updated appropriately.The most recent comment on [this issue](https://github.com/widget-/slack-black-theme/issues/48) addresses the issue.

Alternatively, I've been trying to keep this fork up-to-date with the latest fixes. It also addresses some other stylistic nuances that I haven't seen others address yet. Here it is: https://github.com/d-fay/slack-black-theme#installing-into-slack
